src/datagrid/datagrid.actions.js

```diff
@@ -504,6 +505,28 @@ export const editCellValueValidate = (grid, dataId, keyPath, value, validators =
           const params = validator.params ? Object.values(validator.params) : [];
           validationState = validator.validateWithRowData(value, rowData, ...params);
         }
+      } else if (validator.validateWithGridData) {
+        // Custom validator function with rowData and gridData as parameters
```

Comment:
Is validator.validateWithGridData something what you can give as a prop? Those are not mentioned in readme.